src/ui/atoms/typography.svelte

```diff
@@ -0,0 +1,124 @@
+<script module>
+  import { type Snippet } from "svelte";
+
+  type Size =
+    | "heading-2xl"
+    | "heading-xl"
+    | "heading-lg"
+    | "heading-md"
+    | "body-base"
+    | "body-small"
+    | "label-button"
+    | "label-default"
+    | "caption-default"
+    | "caption-link";
+
+  type Props = {
+    size?: Size;
+    children?: Snippet;
+  };
+</script>
+
+<script lang="ts">
+  const { children, size = "body-base" }: Props = $props();
+</script>
+
+<span class="rcb-typography rcb-typography-{size}">
+  {@render children?.()}
+</span>
+
+<style>
+  /* svelte-ignore css-unused-selector */
```

Comment:
1. Remove original variables in favor of `Typography` component.
2. Review font boldness: `font-weight` and `font-family`.